### PR TITLE
update Simplified-Chinese translation (lang_zh.txt)

### PR DESCRIPTION
**Pull request by Emphasia**
continuing https://github.com/VictorPhilipp/Cities-Skylines-Traffic-Manager-President-Edition/issues/83
https://github.com/Emphasia/TMPE-zh/blob/6476cd2/lang_zh.txt
update based on VictorPhilipp/Cities-Skylines-Traffic-Manager-President-Edition:TLM/TLM/Resources/lang.txt@2f70dca

### CODE REVIEW

**Comment by Emphasia** [comment]
The existing translation is mine and this is an update.
I created an [PR](https://github.com/VictorPhilipp/Cities-Skylines-Traffic-Manager-President-Edition/pull/238) in VictorPhilipp(LinuxFan)'s fork, but ... 😂
Anyway, hope that you can include this into the next patch. ❤

**Comment by Emphasia** [comment]
The existing translation is mine and this is an update.
I created a [PR](https://github.com/VictorPhilipp/Cities-Skylines-Traffic-Manager-President-Edition/pull/238) in VictorPhilipp(LinuxFan)'s fork, but ... 😂
Anyway, hope that you can include this into the next patch. ❤

File: TLM/TLM/Resources/lang_zh.txt [hunk 1]
```diff
@@ -136,92 +136,93 @@ Entering_vehicle 进入车辆
...
-Compact_main_menu Compact main menu
-Window_transparency Window transparency
-Overlay_transparency Overlay transparency
-Remove_this_citizen Remove this citizen
-Show_error_message_if_a_mod_incompatibility_is_detected Show error message if a mod incompatibility is detected
-Remove_parked_vehicles Remove parked vehicles
-Node_is_level_crossing This junction is a level crossing.\nYou cannot disable traffic lights here.
\ No newline at end of file
+Vehicles_follow_priority_rules_at_junctions_with_timed_traffic_lights 优先通行规则适用于带自定时红绿灯的路口
+
```
Comment (krzychu124) [inline TLM/TLM/Resources/lang_zh.txt]: Is this empty line necessary? I can't see it inside other lang files 😉

Comment (Emphasia) [reply TLM/TLM/Resources/lang_zh.txt]: this empty line is not necessary, I just put it there to separate parts.
(and it should not cause any problems, but if it did, just delete it.)

**Pull request merged**